backend/tests/integration/conftest.py

```diff
@@ -42,7 +44,16 @@ def event_loop():
 
 
 @pytest.fixture(scope="module")
-async def db() -> AsyncGenerator[InfrahubDatabase, None]:
+async def db(
```

Comment:
we shouldn't have to redefine it here since it's already in the main `conftest.py` file